tests/pipelines/test_vote.py

```diff
@@ -1,6 +1,7 @@
 from unittest.mock import patch
 
 import rethinkdb as r
+import time
```

Comment:
as per [PEP 8](https://www.python.org/dev/peps/pep-0008/#imports), the grouping of imports is as follows:

> Imports should be grouped in the following order:
>
> 1. standard library imports
> 2. related third party imports
> 3. local application/library specific imports
>
> You should put a blank line between each group of imports.